SlackAPI/Block.cs

```diff
@@ -46,6 +46,12 @@ public class ContextBlock : IBlock
         public string block_id { get; set; }
         public IElement[] elements { get; set; }
     }
+    public class HeaderBlock : IBlock
+    {
+	    public string type { get; } = BlockTypes.Header;
```

Comment:
Indentation is weird here 😛

Reply:
It's so weird because it doesn't show these bad indentations in VS locally for me 😠

Reply:
Yeah, if you're on desktop and click and drag on the whitespace to select it, it looks like there's a tab character and 4 spaces mixed together on this line. The others are all spaces.

Pretty sure visual studio show the tab keys as spaces in the editor 😛

Reply:
Took a few commits, but I think it might be fixed now.

Reply:
Well.

You're not wrong. 🤣

It is now the same for all of the properties listed here. But, it's still got a mix of tabs and spaces.

Can you try selecting the whitespace on line 57, the whitespace before `public string type { ...` and just pasting it in-place of the whitespace on these properties?

If you select the text in GitHub you should be able to see what I mean by spaces vs tabs, a tab character has a width of 6 spaces for a single character on GitHub (on my Chrome, at least), and then there's the regular 4 spaces right after the tab character. It should be 8 spaces, not 4 + 1 tab character (6 spaces).

It probably looks fine locally in VS because Visual Studio displays a single tab character as 4 spaces

If you can't fix it, I can fix it later, it's really more of a meta thing anyways

Reply:
I feel dumb but I just can't seem to fix this. I think VS is doing something with the spaces no matter what I do.